test/e2e/toolchaincluster_test.go

```diff
@@ -30,15 +29,14 @@ func TestToolchainClusterE2E(t *testing.T) {
 // in the target cluster type operator
 func verifyToolchainCluster(t *testing.T, await *wait.Awaitility, otherAwait *wait.Awaitility) {
 	// given
-	current, ok, err := await.GetToolchainCluster(t, otherAwait.Type, otherAwait.Namespace, nil)
+	current, ok, err := await.GetToolchainCluster(t, otherAwait.Namespace, nil)
 	require.NoError(t, err)
 	require.True(t, ok, "ToolchainCluster should exist")
 
-	t.Run("create new ToolchainCluster with correct data and expect to be ready for cluster type "+string(await.Type), func(t *testing.T) {
+	t.Run("create new ToolchainCluster with correct data and expect to be ready for cluster", func(t *testing.T) {
```

Comment:
let's somehow distinguish between the two execution of this function:
  ```suggestion
	t.Run(fmt.Sprintf("create new ToolchainCluster based on '%s' with correct data and expect to be ready", await.ClusterName), func(t *testing.T) {
```

Reply:
done